### DISCUSSION TITLE
Trying to get spamfolder etc but no luck

### DISCUSSION THREAD

**Discussion by sm0tsc** [rspamd/rspamd#3904] [Q&A]
I am tryng this guide.. But folder seems not created.. 

https://rspamd.com/doc/tutorials/feedback_from_users_with_IMAPSieve.html

Anyone have some pointers

//Johan

**Comment by moisseev** [comment]
In this guide you are supposed to create the folders by hand:
https://rspamd.com/doc/tutorials/feedback_from_users_with_IMAPSieve.html#folders

Alternatively you can extend the Sieve scripts to create the folders automatically.

**Comment by sm0tsc** [comment]
Ok
Understand.. So If I want to have it for multiple users I must extend the
script aswell?

//Johan

Den tis 21 sep. 2021 kl 20:18 skrev Alexander Moisseev <
***@***.***>:

**Comment by sm0tsc** [comment]
Maybe I misunderstand something. What I want to achieve is some folders
where users can autotrain rspamd with ham and spam.. Is there another way
to do it?

//Johan

Den tis 21 sep. 2021 kl 20:18 skrev Alexander Moisseev <
***@***.***>:

**Comment by moisseev** [comment]
You probably need this: https://doc.dovecot.org/configuration_manual/howto/antispam_with_sieve/
